components/Events/Footer.vue

```
@@ -225,36 +226,22 @@ const groupedButtonsByType = computed(
 );
 
 function goToPreScreeningOrSchedule(eventSlug: string) {
-  if (eventConfig?.preScreening?.disabled === true) {
-    // Triagem desativada explicitamente → pula direto
-    navigateTo(`/event/${eventSlug}/schedules`);
+  const userStore = useUserStore();
+  const userId = userStore.user?.id;
+
+  if (!userId) {
+    console.warn("Usuário não identificado, não é possível verificar localStorage.");
```

ESLint

[error] Replace `"Usuário·não·identificado,·não·é·possível·verificar·localStorage."` with `⏎······"Usuário·não·identificado,·não·é·possível·verificar·localStorage.",⏎····`

(prettier/prettier)